### PR TITLE
members and teams endpoints

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
currently added endpoint for invitation to orgs
and teams management

### CODE REVIEW

File: playbooks/roles/gitcontrol/files/github_api.py [hunk 1]
```diff
@@ -71,7 +77,8 @@ def update_branch_protection(github_api, owner, repo_name, repo):
     rules = repo[repo_name]['protection_rules']
     if isinstance(rules, str):
         rules = {repo[repo_name]['default_branch']:
-                 read_yaml_file(f'/home/zuul/src/github.com/opentelekomcloud-infra/system-config/playbooks/templates/github/{rules}.yaml')}
+            read_yaml_file(
+                f'{template_folder}/{rules}.yaml')}
```
Comment (gtema) [inline playbooks/roles/gitcontrol/files/github_api.py:81]: We need to have full path here (where SC is being checked out. Alternative is to move rules under the orgs (at least from repo POV)

File: playbooks/roles/gitcontrol/files/github_api.py [hunk 2]
```diff
@@ -18,6 +20,10 @@
 def read_yaml_file(path, org=None, endpoint=None, repo_name=None):
     if endpoint in ['manage_collaborators', 'branch_protection', 'options']:
         path += f'{org}/repositories/{repo_name}.yml'
+    if endpoint in ['teams']:
+        path += f'/{org}/teams/members.yml'
```
Comment (gtema) [inline playbooks/roles/gitcontrol/files/github_api.py:24]: please get rid of the starting "/" char. This explicitly hints that this must be a relative path

**Review by gtema — CHANGES_REQUESTED** [review]

File: playbooks/roles/gitcontrol/files/github_api.py (hunk 1 above)
Comment (anton-sidelnikov) [reply playbooks/roles/gitcontrol/files/github_api.py:81]: Yes, it stored above in variable template_folder = '/home/zuul/src/github.com/opentelekomcloud-infra/system-config/playbooks/templates/github'

Comment (gtema) [reply playbooks/roles/gitcontrol/files/github_api.py:81]: ah, missed that

**Review by gtema — APPROVED** [review]

**Comment by gtema** [comment]
recheck

**Comment by gtema** [comment]
recheck

**Pull request merged**